packages/gatsby-plugin-cms/src/gatsby-node.ts

```diff
@@ -16,11 +9,19 @@ import {
   loadSchema,
   sourceAllNodes,
 } from 'gatsby-graphql-source-toolkit'
+import { outputJSON, pathExists } from 'fs-extra'
+import type { JSONSchema6 } from 'json-schema'
+import type {
+  CreatePagesArgs,
+  PluginOptionsSchemaArgs,
+  SourceNodesArgs,
+} from 'gatsby'
 
+import { sourceAllLocalNodes } from './node-api/sourceAllLocalNodes'
 import type { ContentTypes, Schemas } from './index'
 
 // VTEX IO workspace
-const WORKSPACE = 'master'
+const WORKSPACE = process.env.GATSBY_VTEX_IO_WORKSPACE ?? 'master'
```

Comment:
instead of being a variable, shouldn't this be a parameter of the plugin's options? So when you configure the `gatsby-plugin-cms` plugin in the store's gatsby-config, you would pass which workspace. That way, we would have only one data source and better traceability.

Reply:
I think so. I'll change it :)